### DISCUSSION TITLE
Imagemagick is creating three tiled images from pdfs encoded with acrobat si

### DISCUSSION THREAD

**Discussion by asmartinez** [ImageMagick/ImageMagick6#230] [General]
### ImageMagick version

6.xx

### Operating system, version and so on

ubuntu 20.04

### Description

Is there some rule in Imagemagick that is causing this to happen with PDF created with acrobat distiller?

<img width="363" alt="image" src="https://user-images.githubusercontent.com/22667121/228387142-88b6d74c-c9b7-4e07-8d70-c8c6588b066d.png">

**Comment by fmw42** [comment]
Post your original PDF and your command line

Note there is a separate forum for Imagemagick 6 issues.

**Comment by asmartinez** [comment]
Original pdf is  https://repositorio.colmex.mx/downloads/z603r043c?

Not sure about command line. Ill post in a bit..
